packages/core/src/sse-writer.ts

```diff
@@ -45,6 +46,14 @@ export function createSSEStream(options?: { onClose?: () => void }): {
         end();
       }
     },
+    writeComment(comment: string): void {
```

Comment:
how do clients treat this? do they freak out at non-jsonrpc messages? are we allowed to send just random events down the sse stream that have no bearing on the messages back and forth between client and server?